src/main/java/database/controller/FileDatabaseExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package database.controller;
+
+import database.exception.BadRequestException;
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+public class FileDatabaseExceptionHandler {
+
+    public void register(Javalin app) {
+        FileDatabaseExceptionHandler fileDatabaseExceptionHandler = new FileDatabaseExceptionHandler();
```

Comment:
I don't think we need to create instance of it's own class here. Instead, you can use `this:`: in method reference.